pages/wishes/index.vue

```diff
@@ -122,6 +122,15 @@ export default {
     },
     avatarColor(fullName) {
       return randomMC.getColor({ text: fullName })
+    },
+    wishDelete(wish){
+      this.deleteWish({
```

Comment:
Using `this.$store.dispatch('deleteWish', payload)` is more obvious here, because this action name repeats naming of action.